bsp/boards/openmote-b/board.c

```diff
@@ -183,9 +278,9 @@ static void clock_init(void) {
     /* Set the system clock to 32 MHz */
     SysCtrlClockSet(true, false, SYS_CTRL_SYSDIV_32MHZ);
 
-    /* Set the IO clock to operate at 16 MHz */
+    /* Set the IO clock to operate at 32 MHz */
     /* This way peripherals can run while the system clock is gated */
-    SysCtrlIOClockSet(SYS_CTRL_SYSDIV_16MHZ);
+    SysCtrlIOClockSet(SYS_CTRL_SYSDIV_32MHZ);
```

Comment:
why did you make this change?

Reply:
I just used an replaced the board.c file with the one from the subghz project by Jonathan